eng/Versions.props

```diff
@@ -38,7 +38,7 @@
   -->
   <ItemGroup>
     <!-- Targeting packs are only patched in extreme cases. -->
-    <ProjectServicingConfiguration Include="Microsoft.NETCore.App.Ref" PatchVersion="1" />
+    <ProjectServicingConfiguration Include="Microsoft.NETCore.App.Ref" PatchVersion="2" />
```

Comment:
We should change this comment. `Targeting packs are only patched in extreme cases`

How about `Targeting packs are only patched when we need to patch reference assemblies, intellisense, or source generators`

Reply:
I'll update it.